crates/cargo-wdk/src/actions/package/package_task.rs

```diff
@@ -409,62 +392,55 @@ impl<'a> PackageTask<'a> {
             "SHA256",
             &driver_binary_file_path,
         ];
-
         if let Err(e) = self.command_exec.run("signtool", &args, None) {
             return Err(PackageTaskError::DriverBinarySignCommand(e));
         }
-
         std::result::Result::Ok(())
     }
 
     fn run_signtool_verify(&self, file_path: &Path) -> std::result::Result<(), PackageTaskError> {
         info!(
-            "Verifying {} using signtool",
+            "Verifying {} using signtool.",
             file_path
                 .file_name()
                 .expect("Unable to read file name from the path")
                 .to_string_lossy()
         );
         let driver_binary_file_path = file_path.to_string_lossy();
         let args = ["verify", "/v", "/pa", &driver_binary_file_path];
-
         // TODO: Differentiate between command exec failure and signature verification
         // failure
         if let Err(e) = self.command_exec.run("signtool", &args, None) {
             return Err(PackageTaskError::DriverBinarySignVerificationCommand(e));
         }
-
         std::result::Result::Ok(())
     }
 
     fn run_infverif(&self) -> Result<(), PackageTaskError> {
-        info!("Running InfVerif command");
+        info!("Running InfVerif.");
+        let mut args = vec![
+            "/v",
+            match self.driver_model {
+                DriverConfig::Kmdf(_) | DriverConfig::Wdm => "/w",
+                // TODO: This should be /u if WDK <= GE && DRIVER_MODEL == UMDF, otherwise it should
+                // be /w
+                DriverConfig::Umdf(_) => "/u",
+            },
+        ];
         let additional_args = if self.sample_class {
             let wdk_build_number = self.wdk_build_provider.detect_wdk_build_number()?;
             if MISSING_SAMPLE_FLAG_WDK_BUILD_NUMBER_RANGE.contains(&wdk_build_number) {
                 debug!(
-                    "Skipping InfVerif. InfVerif in WDK Build {} is bugged and does not contain \
-                     the /samples flag.",
-                    wdk_build_number
+                    "Skipping InfVerif for samples class. InfVerif in WDK Build \
+                     {wdk_build_number} is bugged and does not contain the /samples flag."
                 );
+                info!("Skipping InfVerif for samples class. WDK Build: {wdk_build_number}");
```

Comment:
We will end up printing this message twice at the debug level. One solution is to change the `debug!` call above to only mention "InfVerif in WDK Build {} is bugged and does not contain the /samples flag" and let this `info!` call say whatever it is saying right now.